client/state/jetpack-connect/test/actions.js

```diff
@@ -261,26 +260,46 @@ describe( 'actions', () => {
 				} );
 			} );
 
-			it( 'should dispatch sites receive action when request completes', () => {
+			it( 'should dispatch authorize receive site list action when request completes', () => {
```

Comment:
Nitpicking here: it's a `site` receive, not a `site list` receive action, right?